docs/docs/APIdocs/layers/nn/Loss Functions/MultiCriterion.md

```diff
@@ -0,0 +1,71 @@
+## MultiCriterion ##
+
+**Scala:**
+```scala
+val module = MultiCriterion()
```

Comment:
is it better to rename to criterion container or something else meaningful ? Actually it's not a module